cmake/common.cmake

```diff
@@ -607,13 +607,20 @@ function(add_e2e_test)
   )
 
   if (BUILD_END_TO_END_TESTS)
+    if(PBFT)
```

Comment:
If you add this logic to `CCF_NETWORK_TEST_ARGS`, you won't have to duplicate it.